### PR TITLE
Removes Nik from the owners

### PR DESCRIPTION
**Pull request by nlopin**
❤️ My time has come

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-* @nlopin @martimalek @div-Leo @jonotrujillo @arturmiglio
+* @martimalek @div-Leo @jonotrujillo @arturmiglio
```
Comment (martimalek) [inline CODEOWNERS:1]: 🥹 
Let's remove Leo and add Nelson as well 🙈 😭 
```suggestion
* @martimalek @jonotrujillo @arturmiglio @dornelasnelson  
```

**Review by martimalek — APPROVED** [review]

**Comment by snapsnapturtle** [comment]
🥹

**Comment by lukahartwig** [comment]
🫡

**Pull request merged**